examples/2d/2d_gizmos.rs

```diff
@@ -128,6 +128,14 @@ fn draw_example_collection(
         Vec2::from_angle(sin / -10. + PI / 2.) * 50.,
         Color::YELLOW,
     );
+
+    {
+        // We can create more complex arrows using the arrow builder.
+        let mut arrow_builder =
+            gizmos.arrow_2d(Vec2::ZERO, Vec2::from_angle(sin / -10.) * 50., Color::GREEN);
+        arrow_builder.with_double_end();
+        arrow_builder.with_tip_length(10.);
+    };
```

Comment:
```suggestion
    // You can create more complex arrows using the arrow builder.
    let mut arrow_builder =
        gizmos.arrow_2d(Vec2::ZERO, Vec2::from_angle(sin / -10.) * 50., Color::GREEN);
    arrow_builder.with_double_end();
    arrow_builder.with_tip_length(10.);
```
1. Remove the unecessary brackets;
2. NIT: Change the comment to match the existing comments narrative;